doc/stdlib/index-list.html.template

```diff
@@ -287,6 +287,7 @@ through the <tt>Require Import</tt> command.</p>
     theories/Numbers/Cyclic/Int63/Cyclic63.v
     theories/Numbers/Cyclic/Int63/PrimInt63.v
     theories/Numbers/Cyclic/Int63/Int63.v
```

Comment:
Should I move this to the hidden files?

Reply:
This file might not have the best documentation, but its content is meant to be public, so I would not hide it.